docs/UserGuide.md

```diff
@@ -115,9 +115,9 @@ Format: `list`
 
 ### Editing a person : `edit`
 
-Edits an existing client's parameter in FAPro.
+Edit a parameter in existing contacts, such as name, address, phone number, email, occupation, appointment date and tags.
```

Comment:
Just a minor issue. Could u add a comma before "and"? 
Thank you.

Reply:
done!